From 39c1efb5003210b5db6cde831da38386f7be74e4 Mon Sep 17 00:00:00 2001
From: Geetha sowjanya <gakula@marvell.com>
Date: Thu, 6 Jun 2019 12:14:21 +0530
Subject: [PATCH 0174/1921] octeontx2: Fix mbox driver compilation dependency.

Mbox driver needs to get compiled independently of whether AF/PF
driver enabled/disabled. This patch fixes that dependency.

Change-Id: I98445130cd1120e2ba3ab017eb90f74398a80249
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/Makefile b/drivers/net/ethernet/marvell/octeontx2/Makefile
index 4e7d6117b78d..563dc9cd6b36 100644
--- a/drivers/net/ethernet/marvell/octeontx2/Makefile
+++ b/drivers/net/ethernet/marvell/octeontx2/Makefile
@@ -3,5 +3,6 @@
 # Makefile for Marvell OcteonTX2 device drivers.
 #
 
+obj-$(CONFIG_OCTEONTX2_MBOX) += af/
 obj-$(CONFIG_OCTEONTX2_AF) += af/
 obj-y			   += nic/
-- 
2.31.1

